splice_machine/src/test/java/com/splicemachine/derby/utils/SpliceAdminIT.java

```diff
@@ -248,54 +249,59 @@ public void testGetExecInfo() throws Exception {
         TestUtils.FormattedResult fr = TestUtils.FormattedResult.ResultFactory.convert("call SYSCS_UTIL.SYSCS_GET_EXEC_SERVICE_INFO()", rs);
         System.out.println(fr.toString());
-        assertTrue(fr.size()>=1);
+        assertTrue(fr.size() >= 1);
         DbUtils.closeQuietly(rs);
     }
 
     @Test
     public void testGetCacheInfo() throws Exception {
-        CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_CACHE_INFO()");
-        ResultSet rs = cs.executeQuery();
-        while(rs.next()){
-            double hitRate = rs.getDouble("HitRate");
-            double missRate = rs.getDouble("MissRate");
-            assertTrue((hitRate >= 0.0) && (hitRate <= 1.0));
-            assertTrue(missRate <= 1-hitRate + .0001 && missRate >= 1-hitRate - .0001);
+        try( CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_CACHE_INFO()") ) {
+            ResultSet rs = cs.executeQuery();
+            while (rs.next()) {
+                double hitRate = rs.getDouble("HitRate");
+                double missRate = rs.getDouble("MissRate");
+                assertTrue((hitRate >= 0.0) && (hitRate <= 1.0));
+                assertTrue(missRate <= 1 - hitRate + .0001 && missRate >= 1 - hitRate - .0001);
+            }
+            DbUtils.closeQuietly(rs);
         }
-        DbUtils.closeQuietly(rs);
     }
 
     @Test
     public void testGetTotalCacheInfo() throws Exception {
-        CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_TOTAL_CACHE_INFO()");
-        ResultSet rs = cs.executeQuery();
-        rs.next();
-        double hitRate = rs.getDouble("HitRate");
-        double missRate = rs.getDouble("MissRate");
-        assertTrue((hitRate >= 0.0) && (hitRate <= 1.0));
-        assertTrue(missRate <= 1-hitRate + .0001 && missRate >= 1-hitRate - .0001);
-        DbUtils.closeQuietly(rs);
+        try(CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_TOTAL_CACHE_INFO()");
+            ResultSet rs = cs.executeQuery() ) {
+            rs.next();
+            double hitRate = rs.getDouble("HitRate");
+            double missRate = rs.getDouble("MissRate");
+            assertTrue((hitRate >= 0.0) && (hitRate <= 1.0));
+            assertTrue(missRate <= 1 - hitRate + .0001 && missRate >= 1 - hitRate - .0001);
+            DbUtils.closeQuietly(rs);
+        }
     }
 
     @Test
     public void testGetSessionInfo() throws Exception {
-        CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_SESSION_INFO()");
-        ResultSet rs = cs.executeQuery();
-        rs.next();
-        String hostname = rs.getString("HOSTNAME");
-        int session = rs.getInt("SESSION");
-        DbUtils.closeQuietly(rs);
+        String hostname = ""; int session = 0;
+        try(CallableStatement cs = methodWatcher.prepareCall("call SYSCS_UTIL.SYSCS_GET_SESSION_INFO()") ) {
+            ResultSet rs = cs.executeQuery();
```

Comment:
While you're at it, maybe you can `try-with-resource` this one as-well.

Reply:
yes sure